packages/desktop/views/dashboard/send-flow/views/SelectRecipientView.svelte

```diff
@@ -42,17 +42,22 @@
     $: selectedNetworkId = selectorOptions[selectedIndex]?.networkId
     $: selectedRecipient = selectorOptions[selectedIndex]?.selectedRecipient
 
-    let hasNetworkRecipientError: boolean = false
-    $: {
+    let hasInsufficientFunds: boolean = false
```

Comment:
For local variables, the type is inferred automatically by TS.
```suggestion
    let hasInsufficientFunds = false
```